server/controllers/forum.js

```diff
@@ -0,0 +1,213 @@
+const config = require('../config');
+const logger = require('../logger');
+const gh = require('../models/gh');
+const archive = require('../models/archive');
+
+const Render = require('../render');
+const render = Render.render;
+const dropCache = Render.dropCache; // eslint-disable-line no-unused-vars
+const tokens = config.github.tokens || [];
+
+const getToken = user => user ? user.accessToken : tokens[Math.ceil(Math.random() * tokens.length - 1)];
+
+function onError(req, res, err) {
+    logger.error(err);
+    if (err.statusCode === 404) return get404(req, res);
+
+    res.status(500);
+
+return render(req, res, { view: '500' });
+}
+
+function createIssuePage(req, res) {
+    logger.log('createIssue page');
+
+    return render(req, res, {
+        view: 'page-create-post'
+    });
+}
+
+function createIssue(req, res) {
+    logger.log('createIssue api');
+
+    return gh.createIssue(req.body, req.user)
+        .then(postData => res.status(200).send(postData.body))
+        .catch(err => onError(req, res, err));
+}
+
+function getIndex(req, res) {
+    logger.log('getIndex');
+
+    const token = getToken(req.user);
+
+    Promise.all([
+        gh.getIssues(token, req.query),
+        gh.getLabels({ token })
+    ]).then(responses => {
+        const issuesData = responses[0];
+        const labelsData = responses[1];
+
+        render(req, res, {
+            view: 'page-index',
+            issues: issuesData.issues,
+            pagination: issuesData.pagination,
+            labels: labelsData
+        });
+    }).catch(err => onError(req, res, err));
+}
+
+function getIssues(req, res) {
+    logger.log('getIssues');
+
+    return gh.getIssues(getToken(req.user), req.query)
+        .then(issuesData => render(req, res, {
+            view: 'page-index',
+            issues: issuesData.issues,
+            pagination: issuesData.pagination
+        }, {
+            block: 'issues'
+        }))
+        .catch(err => onError(req, res, err));
+}
+
+function getComplexIssue(req, res) {
+    const id = +req.params.id;
+    const token = getToken(req.user);
+
+    logger.log('getComplexIssue', id);
+
+    return (id > 0 ? Promise.all([
+        gh.getIssue(id, token),
+        gh.getIssueComments(id, { token })
+    ]) : Promise.all([
```

Comment:
line break
we have a different style of line break in each ternary operator, so let it be with line break after `?` and `:`

Reply:
I would say we havn't have any style of line break in ternary. And here line break not in the ternary, but in array. Though --- ok